spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +41,41 @@
       end
     end
   end
+
+  describe "backward compatibility" do
+    let(:notifier) { instance_double("Notiffany::Notifier") }
+
+    before do
+      allow(Notiffany::Notifier).to receive(:new).and_return(notifier)
+    end
+
+    after do
+      Guard::Notifier.instance_variable_set(:@notifier, nil)
+    end
+
+    context "multiple parameters" do
+      before do
+        subject.connect
+        allow(notifier).to receive(:notify).with("A", {priority: 2, image: :failed}).and_return(true)
```

Comment:
Line is too long. [101/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.